sdk/servicebus/test-resources.json

```diff
@@ -117,6 +181,10 @@
     "SERVICEBUS_SECONDARY_NAMESPACE_CONNECTION_STRING": {
       "type": "string",
       "value": "[listkeys(variables('authRuleSecondaryResourceId'), variables('sbVersion')).primaryConnectionString]"
+    },
+    "STORAGE_CLAIM_CHECK_CONNECTION_STRING": {
+      "type": "string",
+      "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(variables('storageAccountId'), providers('Microsoft.Storage', 'storageAccounts').apiVersions[0]).keys[0].value, ';EndpointSuffix=core.windows.net')]"
```

Comment:
Don't do manual connection string generation.

Reply:
Out of curiosity, what's the alternative?  I don't recall there being another way in ARM to query the storage connection strings since they can't be returned via 'listKeys' like many of the other services.

Reply:
there is apparently a function that can do this for you - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/test-resources.json#L647

Reply:
That's... interesting.    Where does that come from?   It's not in the ARM reference docs that I can find:

https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions

Reply:
ahhh... it's a custom function in their template that does the same concat: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/test-resources.json#L510

Reply:
ah I see.. so I suppose there is no silver bullet here.

Reply:
There is, use AAD auth.

Reply:
We typically use connection string in our samples as it is easier to get started.

Reply:
At least don't hardcode the `EndpointSuffix=core.windows.net`